### PR TITLE
new wrap: dragonbox

### PR DESCRIPTION
**Pull request by HassanSajjad-302**
Added https://github.com/jk-jeon/dragonbox/

### CODE REVIEW

**Comment by ilmari-lauhakangas** [comment]
Ping @jpakkane

File: releases.json [hunk 1]
```diff
@@ -1660,5 +1660,13 @@
       "1.3.3-2",
       "1.3.3-1"
     ]
+  },
+  "dragonbox": {
+    "dependency_names": [
+      "dragonbox"
+    ],
+    "versions": [
+      "1.1.2"
```
Comment (jpakkane) [inline releases.json:1669]: This needs to be `1.1.2-1`. The number after the dash signifies the "wrapdb version". There may be several for any given upstream release.

File: subprojects/packagefiles/dragonbox/meson.build [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    'cpp',
+    version : '1.1.2',
+    license : 'Boost',
+    default_options : ['cpp_std=c++17']
+)
+
+dragonbox_lib = library('dragonbox', 'source/dragonbox_to_chars.cpp', include_directories : include_directories('include/'))
+
+dragonbox_dep = declare_dependency(
+    include_directories : include_directories('include/'),
```
Comment (jpakkane) [inline subprojects/packagefiles/dragonbox/meson.build:11]: This should not have the trailing slash.

**Review by jpakkane — CHANGES_REQUESTED** [review]

**Comment by HassanSajjad-302** [comment]
Changes Requested Applied.

**Comment by jpakkane** [comment]
You need to set the C++ version in `ci_config.json`. Due to reasons it defaults to empty and on the compiler we use for running CI it means either C++11 or something even older.

In addition the list of items in the JSON must be sorted, so you need to add the entry to the correct place, not at the end of the array.

**Comment by HassanSajjad-302** [comment]
Updated.

**Pull request merged**